packages/client/src/object/convertWireToOsdkObjects.ts

```diff
@@ -197,11 +197,11 @@ export async function convertWireToOsdkObjects2(
  * Utility function that lets us take down selected property names from an interface
  * and convert them to an array of property names on an object.
  */
-function convertInterfacePropNamesToObjectPropNames(
-  objectDef: FetchedObjectTypeDefinition & { interfaceMap: {} },
+export function convertInterfacePropNamesToObjectPropNames(
```

Comment:
Since this wasn't exported before, can we mark it as `@internal` so we don't accidentally leak it?